### PR TITLE
Update database.rst

### PR DESCRIPTION
**Pull request by nitish705702**
### Description

Fixes #<Issue Number>

### CODE REVIEW

**Comment by nabobalis** [comment]
Hi @nitish705702, can you explain your thought process behind these changes? 

Looking at them at a glance, I can see some changes that I do not think are restructuredtext syntax. Also I do not understand the choice to add the HelioViewer Changes into the database.rst file.  

For reference here is a guide on RST I still load up: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

File: docs/guide/acquiring_data/database.rst [hunk 1]
```diff
@@ -55,11 +53,13 @@ can also be committed explicitly using the :meth:`Database.commit` method.
...
-
-2. Adding new entries
+________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+2. <Adding new entries>
 ---------------------
-Each entry in a database is an instance of the class
-:class:`tables.DatabaseEntry` with the following attributes:
+             Each entry in a database is an instance of the class
+             :class:tables.DatabaseEntry (
+                  https://docs.sunpy.org/en/latest/api/sunpy.database.tables.DatabaseEntry.html#sunpy.database.tables.DatabaseEntry
```
Comment (nitish705702) [inline docs/guide/acquiring_data/database.rst]: Adding url of the data base entry file

File: docs/guide/acquiring_data/database.rst [hunk 2]
```diff
@@ -800,3 +791,196 @@ to 10 and therefore removes the 5 entries that been used least recently.
      18    2011-06-07 20:37:52 ...               19.5               19.5
      58    2011-06-07 06:33:29 ... 17.400000000000002 17.400000000000002
      59    2011-06-06 00:00:00 ...                N/A                N/A
+****************************************************************************************************************************************
```
Comment (nitish705702) [inline docs/guide/acquiring_data/database.rst]: Let's add a file to a new file, and the name of file is 'QueryingHelioviewer.org_with_sunPy'  .
which is helped for 'SunPy can be used to make several basic requests using the The Helioviewer.org API including generating a PNG screenshot and downloading a JPEG 2000 image.'  likely related problem.

File: docs/guide/acquiring_data/database.rst (hunk 1 above)
Comment (nabobalis) [reply docs/guide/acquiring_data/database.rst]: So the code at line 60 here, or the old line 62, already (at least it should) embeds a link to that when the documentation is built. So we don't have to do it manually.

File: docs/guide/acquiring_data/database.rst (hunk 2 above)
Comment (nabobalis) [reply docs/guide/acquiring_data/database.rst]: What related problem?

We already have all of this text in the helioviewer.rst file, so I am still confused as to its inclusion here.

Comment (nitish705702) [reply docs/guide/acquiring_data/database.rst]: here my mistake ,sorry.

Comment (nabobalis) [reply docs/guide/acquiring_data/database.rst]: That is no problem. If you do need any help or advice, you can ask!

**Pull request closed without merging**